automation-packages/load-testing-okhttp/src/main/resources/automation-package.yaml

```diff
@@ -8,9 +8,17 @@ plans:
         users: 5
         iterations: 10
         children:
-          - callKeyword:
-              keyword: "OpenCart home"
+          - sequence:
+              children:
+              - callKeyword:
+                 keyword: "okhttp Init"
```

Comment:
As mentioned above, I wouldn't call this KW explicitly but do the initialization in the Keywords implicitly